src/i18n/strings/en_EN.json

```diff
@@ -3120,6 +3120,9 @@
         "empty": "You don't currently have any stickerpacks enabled",
         "empty_add_prompt": "Add some now"
     },
+    "tac": {
```

Comment:
A `threads_activity_centre` category is available in the lang json

Reply:
oops, thanks